Marlin/src/module/stepper/trinamic.cpp

```diff
@@ -70,7 +70,7 @@ enum StealthIndex : uint8_t { STEALTH_AXIS_XY, STEALTH_AXIS_Z, STEALTH_AXIS_E };
 // Stepper objects of TMC2130/TMC2160/TMC2660/TMC5130/TMC5160 steppers used
 int findDriver(int driverType, int driverIndex)
 {
-  for (int i = 0; i < NUM_SPI_DEVICES; i++)
+  for (u_int8_t i = 0; i < NUM_SPI_DEVICES; i++)
```

Comment:
`uint8_t` ?

Reply:
no matter how big your board is NUM_SPI_DEVICES shouldn't be int-sized.
255 spi devices should be "enough"...i think.

Reply:
Right. Any **existing** integer type would be ok.

Reply:
u_int8_t  doesn't exist on your end? :open_mouth:
![image](https://user-images.githubusercontent.com/897723/70044244-485b4080-15c2-11ea-9185-d2ca41f1ab0b.png)

Reply:
At least u_int8_t is unusual in Marlin.
![image](https://user-images.githubusercontent.com/17075493/70047844-2c0ed200-15c9-11ea-9130-c7d6ecf3799f.png)

Reply:
unusual but **existing**
anyway I changed it so no one gets shocked.